node/nodejs_intro/README.md

```diff
@@ -140,23 +196,29 @@ What is expected output?
     Yoda
     I am
 </details>
+<p>&nbsp;</p>
 
-Let's take a look at why? The first thing that will be moved onto our stack is `console.log("Hello,");`. This immediately resolve and be popped off our stack.
+Let's take a look at why:
+>The first thing that will be moved onto our stack is  `console.log("Hello,");`. This immediately resolves and is popped off our stack.
 
-Next thing pushed onto the stack is the setTimeout. Because this is an asynchronous call it will get moved over from our stack to the WebAPI and begin to count down for 1 second.
+>The next thing pushed onto the stack is the `setTimeout`. Because this is an `asynchronous` call it will get moved over from our stack to the WebAPI and begin to count down for 1 second.
 
-Our code continues to run and pushes the final console log onto our empty stack. It immediately resolves and is then popped off.
+>Our code continues to run and pushes the final `console.log` onto our empty stack. This immediately resolves and is then popped off the call stack.
 
-The current state of our loop is: Empty Stack, SetTimeout in WebAPI, and empty callback queue.
+>At this point, the state of our loop is: `Empty Stack, SetTimeout in WebAPI, and empty callback queue.`
 
-After 1 second, our setTimeout resolves and moves the callback (our final console log) to the callback queue.
+>After 1 second, our `setTimeout` resolves and moves the callback (our final `console.log`) to the callback queue.
 
-Because our call stack is empty, the first item of our callback queue is moved onto the stack. The console log is
+>Because our call stack is empty, the first item of our callback queue is moved onto the stack. The `console.log` is
 immediately resolved and popped off the stack.
 
-Our code finishes running.
+>Our code finishes running.
 
-Let's try another example:
+*Still scratching your head?*
```

Comment:
I think we could do with another straightforward example here to give the learners more chances to understand. This could be hidden in a `<details>` block as an extra exercise. For example, a small variation on the previous code:
```js
setTimeout(() => console.log("I am"), 1000);
console.log("Hello,");
console.log("Yoda");
```
And perhaps a question like, 
```
How could we change this code to make the output be `Hello, I am Yoda`?
```

Reply:
👍